interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -179,10 +181,49 @@ namespace {
   };
 } // unnamed namespace
 
+static void HandlePlugins(CompilerInstance& CI,
+                         std::vector<std::unique_ptr<ASTConsumer>>& Consumers) {
+  // Copied from Frontend/FrontendAction.cpp.
+  // FIXME: Remove when we switch to the new cling driver.
```

Comment:
"the new cling driver" -> "a tools-based cling driver"? Maybe my new wording is incorrect, but I'd appreciate if this comment could call out the new design foundation instead of "new", to make it clear what makes this superfluous.